abaplint.json

```diff
@@ -107,7 +107,9 @@
       "pragmas": ["##NO_TEXT"],
       "pseudo": ["#EC NOTEXT"]
     },
-    "unused_types": true,
+    "unused_types": {
+      "exclude": ["zif_abapgit_aff_types_v1", "zif_abapgit_aff_chko_v1"]
```

Comment:
zif_abapgit_aff_chko_v1 should be statically referenced?

Reply:
resolved in eb77cb7 and 59d897, since the aff types are not necessary in this implementation.